install/kubernetes/cilium/values.yaml

```diff
@@ -352,6 +352,8 @@ daemon:
 
 enableCnpStatusUpdates: false
 
+enableK8sEventHandover: false
```

Comment:
Can you also add a small comment for this:

```
Configures the use of the KVStore to optimize Kubernetes event handling by mirroring it into the KVstore for reduced overhead in large clusters.
```

Reply:
Done.